src/ServiceProvider.php

```diff
@@ -41,6 +41,7 @@ public function boot()
 
         $this->publishes([
             $this->fullPath('config/pretty-routes.php') => $this->app->configPath('pretty-routes.php'),
+            $this->fullPath('resources/views/styles.blade.php') => resource_path('views/vendor/pretty-routes/styles.blade.php'),
```

Comment:
The `resource_path()` function should be replaced with `$this->app->resourcePath()` to maintain code identity.

Reply:
```suggestion
            $this->fullPath('resources/views/styles.blade.php') => $this->app->resourcePath('views/vendor/pretty-routes/styles.blade.php'),
```